recipes/scikit-beam/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.0.8" %}
+
+package:
+    name: scikit-beam
+    version: {{ version }}
+
+source:
+    url: https://github.com/scikit-beam/scikit-beam/archive/v{{ version }}.tar.gz
+    fn: scikit-beam-v{{ version }}.tar.gz
+    sha256: 084690482e418defbe36bb0969133d8de22d21b4a06e09c62ccb6b1717d2e2b4
+
+build:
+  number: 0
+  script:
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
```

Comment:
Is this still needed? https://github.com/conda/conda-build/issues/910 was closed.